source/github-helpers/get-default-branch.ts

```diff
@@ -38,7 +38,11 @@ export default cache.function(async (repository: Partial<RepositoryInfo> = getRe
 
 	return response.repository.defaultBranchRef.name;
 }, {
-	maxAge: 10,
-	staleWhileRevalidate: 20,
+	maxAge: {
+		days: 5
```

Comment:
```suggestion
		hours: 1
```

Many features depend on this, it should be as fresh as possible.

`staleWhileRevalidate` still keeps it in the cache a little longer anyway, so features remain instant.